### DISCUSSION TITLE
Name Search - register dependency

### DISCUSSION THREAD

**Discussion by ShadowSix** [rustdesk/rustdesk#1366] [General]
If you rename the client to which you connected, for example: User01, then the search will not be able to find it when writing in a different register, for example, user01 or USER01, it will not give a result

I didn't see any discussion on this. I thought it would be useful:)
![image](https://user-images.githubusercontent.com/20943091/186637166-ffab0dce-13be-4f4b-bdac-a777840f92ab.png)
![image](https://user-images.githubusercontent.com/20943091/186637216-8c8f4a1e-f15e-4e7a-a496-fce1fdcc2524.png)
![image](https://user-images.githubusercontent.com/20943091/186637254-434fc35c-3f77-4b69-99b8-df91ed556375.png)
![image](https://user-images.githubusercontent.com/20943091/186637300-eb000903-b041-4d4c-a4bc-e808122439ab.png)

**Comment by rustdesk** [comment]
and https://github.com/rustdesk/rustdesk/issues/774

**Comment by rustdesk** [comment]
@Kingtous
